R-package/R/xgb.plot.importance.R

```diff
@@ -1,64 +1,75 @@
-#' Plot feature importance as a bar graph
+#' Plot feature importance
 #'
 #' Represents previously calculated feature importance as a bar graph.
-#' \code{xgb.plot.importance} uses base R graphics, while \code{xgb.ggplot.importance} uses the ggplot backend.
+#' - `xgb.plot.importance()` uses base R graphics, while
+#' - `xgb.ggplot.importance()` uses "ggplot".
 #'
-#' @param importance_matrix a \code{data.table} returned by \code{\link{xgb.importance}}.
-#' @param top_n maximal number of top features to include into the plot.
-#' @param measure the name of importance measure to plot.
-#'        When \code{NULL}, 'Gain' would be used for trees and 'Weight' would be used for gblinear.
-#' @param rel_to_first whether importance values should be represented as relative to the highest ranked feature.
-#'        See Details.
-#' @param left_margin (base R barplot) allows to adjust the left margin size to fit feature names.
-#'        When it is NULL, the existing \code{par('mar')} is used.
-#' @param cex (base R barplot) passed as \code{cex.names} parameter to \code{barplot}.
-#' @param plot (base R barplot) whether a barplot should be produced.
-#'        If FALSE, only a data.table is returned.
-#' @param n_clusters (ggplot only) a \code{numeric} vector containing the min and the max range
+#' @param importance_matrix A `data.table` as returned by [xgb.importance()].
```

Comment:
Trying to find the pattern of reference, what's the square bracket? I see that you are using single quote, plain text, square bracket in a mix for various internal references.

Reply:
This is like \link{\code{xgb.importance}}, https://roxygen2.r-lib.org/articles/markdown.html (middle section).